### PR TITLE
requirements.txt added

### PR DESCRIPTION
**Pull request by allaffa**

### CODE REVIEW

**Review by mkstoyanov — CHANGES_REQUESTED** [review]
This removes the .*txt from the .gitignore, but no file has been added.

Adding the `requirements.txt` should be done with either a new commit or a new PR.

I'm betting that you're still using `git add *` which still ignores the extra files and can potentially add temporary files (remember the problem that we had during release).

I had to put in an effort, but I managed to teach myself out of `git add *`. The only commands that you should be allowed to use are:
```
# this goes through all the changes line by line and helps you catch last minute problems
git add -p 
# this adds a specific new file
git add <file>
```

**Comment by allaffa** [comment]
> This removes the .*txt from the .gitignore, but no file has been added.
> 
> Adding the `requirements.txt` should be done with either a new commit or a new PR.
> 
> I'm betting that you're still using `git add *` which still ignores the extra files and can potentially add temporary files (remember the problem that we had during release).
> 
> I had to put in an effort, but I managed to teach myself out of `git add *`. The only commands that you should be allowed to use are:
> 
> ```
> # this goes through all the changes line by line and helps you catch last minute problems
> git add -p 
> # this adds a specific new file
> git add <file>
> ```

Actually, I am using `git add files` after the mishap we had some time ago :) 
The reason why I removed `*.txt` from `.gitignore` is because this PR adds "requirements.txt", but the file is indeed ignored and not tracker unless I remove `*.txt` from `.gitignore`

**Comment by mkstoyanov** [comment]
:) Good point for `git add files`.

My main comment was that the `requirements.txt` was missing.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+matplotlib
+torch>=1.8
```
Comment (mkstoyanov) [inline requirements.txt:2]: This is sufficient for AADL, but the examples have additional requirements as mentioned in the Issue.

We should add another `requirements.txt` for the examples, or maybe even one for each example.

Comment (allaffa) [reply requirements.txt:2]: Good point. Is it good practice to add the additional `requirements.txt` in each directory, or should I add th additional requirement files at the same level as this one?

Comment (mkstoyanov) [reply requirements.txt:2]: I'm not sure what the common practice is, but I think putting things in the corresponding folder would be best.

We don't want users running just AADL to install all the example dependencies, and we don't want a pile of requirements in the main folder like `requirements-example1.txt`, `requirements-example2.txt` and so on.

**Review by mkstoyanov — APPROVED** [review]
Looks good.

**Pull request merged**